src/compiler/diagnosticMessages.json

```diff
@@ -3818,6 +3818,10 @@
         "category": "Suggestion",
         "code": 80001
     },
+    "Did not find a declaration file for module '{0}'. '{1}' implicitly has an 'any' type.": {

```

Comment:
we probably should take your suggestion of converting an error diagnostic to a suggestion diagnostic.